ansible/host_vars/default

```diff
@@ -47,7 +47,7 @@ code_dir: /home/{{ansible_ssh_user}}/Tatoeba
 # Number of revisions to keep in verions directory (for later restore); should be greater than 0
 revision_limit: 3
 git_temp_dir: .temp_repo
-repo_version: HEAD
+repo_version: dev
```

Comment:
```
Setting HEAD here makes it pull the 'master' branch, but there is no such branch.
```

Nit: the change itself is harmless, and probably more explicit. I am mostly curious about commit message. github.com/tatoeba/tatoeba2 has `dev` as default branch, I wonder why `master` would be pulled by default.